thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/models/AnomalyResultModel.js

```diff
@@ -111,8 +111,64 @@ AnomalyResultModel.prototype = {
   },
   updateModelAndNotifyView : function(anomaliesWrapper) {
     this.anomaliesWrapper = anomaliesWrapper;
+    this.formatAnomalies();
     this.renderViewEvent.notify();
   },
+
+  /**
+   * Helper Function that returns formatted anomaly region duration data for UI
+   * @param  [date] start: the anomaly region start
```

Comment:
Parameter types should be in curly braces instead of square brackets, also the suffixed colon `:` isn't part of the JsDoc spec